Jenkinsfile

```diff
@@ -49,6 +49,8 @@ pipeline {
                         SKIP = prDetails.draft.equals(true) || prDetails.labels.count { label -> label.name.equalsIgnoreCase('CI/skip') }.equals(1) || prDetails.labels.count { label -> label.name.equalsIgnoreCase("CI/skip-${PLATFORM}") }.equals(1)
                         RUN_NETWORK_AUDIT = prDetails.labels.count { label -> label.name.equalsIgnoreCase('CI/run-network-audit') }.equals(1)
                         RUN_AUDIT_DEPS = prDetails.labels.count { label -> label.name.equalsIgnoreCase('CI/run-audit-deps') }.equals(1)
+                        STORYBOOK_PR_URL = prDetails.labels.count { label -> label.name.equalsIgnoreCase('CI/storybook-url') }.equals(1)
```

Comment:
you just need RUN_STORYBOOK - if not present, you skip its execution

Reply:
@petemill Are you happy with this behaviour instead of specifically suppressing it? Plus @mihaiplesa this doesn't run the storybook stage, just enables the PR comment or not. So `ADD_STORYBOOK_URL` as an alternative name?

Reply:
I think it's safer to still always run storybook, but only output the URL if asked to. If that's what you mean @itschrishudson? At least for now...
There are other file patterns that could break storybook - dep updates, config changes, etc.

Reply:
Yes @petemill as mentioned on Slack, I don't intend to change the running of storybook, I am only talking about the PR comment functionality, not running storybook or not. That should always run on PRs. My query is just which way do you prefer to provide that option:

Current implementation as you described in the devops PR - A label that can be added automatically or manually and if it is present will add the PR comment. But also including a no-url label that if present would specifically suppress that comment if present.

OR (Mihai's suggestion)

A label that can be automatically or manually be applied and if present will add the PR comment.
PR comment will not be applied if that label is removed or doesn't exist on the PR.

Any preference on those two?

Reply:
Oh, then let's keep running it as before and since you mentioned the comment the PR is only posted once and its content (and link) doesn't change. We can have a generic label like `storybook` indicating changes - this would be applied automatically and then comment on the PR with the link.